ci/scripts/rust_toml_fmt.sh

```diff
@@ -17,5 +17,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# Run cargo-tomlfmt with flag `-d` in dry run to check formatting
+# without overwritng the file. If any error occur, you may want to
+# rerun 'cargo tomlfmt -p path/to/Cargo.toml' without '-d' to fix
+# the formatting automatically.
```

Comment:
I'd also add some documentation to the script to guide the user on how to fix the `Cargo.toml` formatting.